block-template-parts/comments.html

```diff
@@ -0,0 +1,9 @@
+<!-- wp:comments-query-loop -->
+<div class="wp-block-comments-query-loop"><!-- wp:comment-template -->
+	<!-- wp:post-comment-author {"isLink":true} /-->
```

Comment:
> I'm able to place it in manually via the site editor, and that generally seems to work?

I'm confused about how this is expected to work in 5.9. 

The `post-comment-author`, `post-comment-date`, `post-comment-content`,  and `post-comments-form` are all [currently marked as experimental](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-library/src/index.js#L245-L257).

Reply:
Can we use blocks that are marked as experimental, they just don't show up in the inserter?

Reply:
No, that doesn't sound right.

Reply:
Left a comment here to get some clarification: https://github.com/WordPress/gutenberg/pull/35979#issuecomment-956540014